### ISSUE TITLE
Change settings?

### ISSUE THREAD

**Issue by philippeowagner** [thomasbrueggemann/paperless-desktop#17]
Hi Thomas
Where are the settings stored? I've to change the host of my paperless endpoint.

**Comment by thomasbrueggemann** [comment]
Hi, the settings are stored in the localStorage of the WebView https://github.com/thomasbrueggemann/paperless-desktop/blob/master/js/src/components/Settings.js#L70

But you should be able to navigate to the settings-view by clicking on the little "gear" icon in the Button-Bar and change your settings there. As far as I now this is already implemented.

**Comment by philippeowagner** [comment]
@thomasbrueggemann the settings view will not be loaded by clicking on the gear icon.

**Comment by thomasbrueggemann** [comment]
Oh okay. Than you have two options, untill the settings view is properly implemented:

1) You remove and reinstall the application completely
2) You start the app with dev-tools enabled https://github.com/thomasbrueggemann/paperless-desktop/blob/master/main.js#L145 and type in the console ``` localStorage.setItem("settings.host", "http://localhost:1234"); ``` with your new host.

**Comment by philippeowagner** [comment]
The 1st option did not work for me so I tried the 2nd one. 

Getting this error while trying to follow the installation guide provided here: https://github.com/danielquinn/paperless/issues/69#issuecomment-285868344 

```
Engine:paperless-desktop phi$ /Users/phi/.npm-packages/bin/npm install
npm WARN worker-loader@0.7.1 requires a peer of webpack@>=0.9 <2 || ^2.1.0-beta but none was installed.

```

Because it was _just_ a _WARN_ing I tried to ..
 
Engine:paperless-desktop phi$ /Users/phi/.npm-packages/bin/npm start

and got 
```
> paperless-desktop@1.0.0 start /Users/phi/GitHub/paperless-desktop
> gulp && electron .

module.js:327
    throw err;
    ^

Error: Cannot find module 'caniuse-db/features-json/css-featurequeries.json'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/phi/GitHub/paperless-desktop/node_modules/gulp-autoprefixer/node_modules/autoprefixer/lib/supports.js:16:10)
    at Object.<anonymous> (/Users/phi/GitHub/paperless-desktop/node_modules/gulp-autoprefixer/node_modules/autoprefixer/lib/supports.js:255:4)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
``` 

Maybe you could point me to the right direction :) Thanks

**Comment by thomasbrueggemann** [comment]
That looks as if the `npm install` did not run through properly. Try to manually remove the `node_modules` folder within the `paperless-desktop` folder and run `npm install` again in the  `paperless-desktop` folder.

As an alternative, if you just want to use paperless-desktop in its current state: I just packaged a macOS executable as a pre-release. You can use that without having to deal with node and npm. 
https://github.com/thomasbrueggemann/paperless-desktop/releases/tag/v0.0.2-alpha

**Comment by philippeowagner** [comment]
Problem solved 🎉 !! Thank you very much for 0.0.2.

**Comment by philippeowagner** [comment]
BTW: The suggestion [in your comment](https://github.com/thomasbrueggemann/paperless-desktop/issues/17#issuecomment-287532953) did the trick.
